src/lib/pickles/test/optional_custom_gates/pickles_test_optional_custom_gates.ml

```diff
@@ -239,6 +239,53 @@ let main_fixed_lookup_tables () =
        ; (* v3 *) w6 = exists Field.typ ~compute:(fun () -> v3)
        } )
 
+(* Parameters *)
+(* nb of fixed lookup tables *)
```

Comment:
```suggestion
(* number of fixed lookup tables *)
```
Btw, why between 0 and 10? More specifically why 10 as an upper limit and why include 0 as a possibility?

Reply:
No strong reason.
The `0` values are fixed in another PR.

Reply:
Including the fix in this one already.